src/status_im/contexts/wallet/create_account/select_keypair/view.cljs

```diff
@@ -63,16 +63,20 @@
       :details             {:full-name (:name item)
                             :address   (when (zero? index)
                                          (utils/get-shortened-compressed-key compressed-key))}
+      :on-press            #(set-selected-key-uid (:key-uid item))
       :accounts            accounts
-      :default-selected?   (zero? index)
+      :selected?           (= selected-key-uid (:key-uid item))
       :container-style     {:margin-horizontal 20
                             :margin-vertical   8}}]))
 (defn view
   []
   (let [{:keys [compressed-key customization-color]} (rf/sub [:profile/profile])
```

Comment:
nit: is there a smaller sub to get this data  for compressed-key and customization-color?

Reply:
@J-Son89 Nope there isn't. Do we want to have a sub for every combination of 2 or 3 keys? We can end up with a very large number of subs.

Reply:
I think we should definitely not oversubscribe, but you can have multiple subscriptions (checked with @flexsurfer and he's fine with this):

i.e
```
(let [compressed-key (rf/sub [:profile/compressed-key])
       customization-color (rf/sub [:profile/customization-color])]
```
`compressed-key` will never re-run, since it never changes, while `customization-color` might, but at least we are insulated from any profile change.
Also, if you have the time, `profile/profile-with-image` should be it's own subscription, is really heavy and it's just extracting one field:

`(let [profile-picture (rf/sub [:profile/image]) // or any naming that makes sense`
and something on those lines:

```
(re-frame/reg-sub
    :profile/image
    :<- [:profile/profile-with-image]
    (fn [profile]
     (profile.utils/photo profile))
```
Otherwise the whole thing will run any time anything change in profile
thanks!

Reply:
Thanks for your input @cammellos, I will update the subs and the `profile-picture` sub also accordingly